packages/modules/pricing/src/services/pricing-module.ts

```diff
@@ -1021,12 +1072,53 @@ export default class PricingModuleService<
             price_set_id: priceSetId,
             title: "test", // TODO: accept title
             rules_count: numberOfRules,
-            priceRules,
+            price_rules: priceRules,
```

Comment:
Works with `price_rules`, but not with `priceRules`. However, in both the cases, the `CreatePriceDTO` does not this property

Reply:
as discussed we can fix it by changing to snake case as you mentioned